run-lstm-with-bmi.py

```diff
@@ -9,36 +9,37 @@
 import bmi_lstm
 
 # creating an instance of an LSTM model
-print('creating an instance of an BMI_LSTM model object')
+print('Creating an instance of an BMI_LSTM model object')
 model = bmi_lstm.bmi_LSTM()
 
 # Initializing the BMI
 print('Initializing the BMI')
 model.initialize(bmi_cfg_file=Path('./bmi_config_files/01022500_hourly_all_attributes_forcings.yml'))
```

Comment:
I'd recommend putting all manual entries at the top of the file in a clearly marked location.

Reply:
ok, assuming you mean file paths.